tools/osx/macosx_bundle.sh

```diff
@@ -268,7 +270,7 @@ install_name_tool -delete_rpath RawTherapee.app/Contents/Frameworks "${EXECUTABL
 install_name_tool -add_rpath @executable_path "${EXECUTABLE}"-cli
 
 # Codesign the app
-if ! test -z "${CODESIGNID}" ; then
+if [ -n "${CODESIGNID}" ] ; then
```

Comment:
@Benitoite if the script uses Bash, and it does, then it should stick to it, so use the `[[` shell keyword and not the `[` test command.

No need for curly braces.

No space between `]]` (or `]`) and `;`

Reply:
Changed to `[[`, `{}` and `"` removed from strings in `[[` statements, space removed from some `;`